packages/core/test/scp-cf/destination-accessor.spec.ts

```diff
@@ -46,7 +50,7 @@ function mockEnvDestinations() {
 
 const environmentDestinations = [
   {
-    name: 'ErpQueryEndpoint',
+    name: 'TESTINATION',
```

Comment:
:-)

Reply:
I'm proud of you 👍

Reply:
Sorry, I just don't like `ErpQueryEndpoint` ;)

Reply:
The makers of `Bernie_and_cert` bring you now `TESTINATION`